src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/upgrade/upgrade.component.html

```diff
@@ -1,13 +1,49 @@
-<ng-container *ngIf="{upgradeInfo: upgradeInfo$ | async, error: upgradeInfoError$ } as upgrade">
-  <div class="row h-25 ms-1"
-       *ngIf="!upgrade.upgradeInfoError && upgrade.upgradeInfo as upgradeInfo; else checkUpgrade">
-    <ng-container *ngIf="healthData$ | async as healthData">
-      <div class="col-lg-3 h-50 d-flex flex-column border justify-content-center align-items-center">
-        <span class="bold">Current Version</span>
-        <span class="mt-1">{{ version }}</span>
+<div class="row h-25"
+     *cdScope="'configOpt'">
+  <ng-container *ngIf="healthData$ | async as healthData">
+    <cd-card class="col-sm-3 px-3 d-flex"
+             cardTitle="Upgrade Status"
+             i18n-cardTitle
+             aria-label="Upgrade Status"
+             i18n-aria-label
+             id="upgradeStatus">
+      <div class="d-flex flex-column justify-content-center align-items-center"
+           *ngIf="info$ | async as info; else checkingForUpgradeStatus">
+        <ng-container *ngIf="info.versions.length > 0; else noUpgradesAvailable">
+          <div i18n-ngbTooltip
+               [ngbTooltip]="(healthData.mgr_map | mgrSummary).total <= 1 ? 'To upgrade, you need minimum 2 mgr daemons.' : ''">
+            <button class="btn btn-accent mt-2"
+                    id="upgrade"
+                    aria-label="Upgrade now"
+                    (click)="upgradeNow(info.versions[info.versions.length - 1])"
+                    [disabled]="(healthData.mgr_map | mgrSummary).total <= 1"
+                    i18n>Upgrade to {{ info.versions[info.versions.length - 1] }}</button>
+          </div>
+          <a class="mt-2 link-primary mb-2"
+             (click)="startUpgradeModal()"
+             i18n>Select another version...</a>
+        </ng-container>
+      </div>
+    </cd-card>
+
+    <cd-card class="col-sm-3 px-3 d-flex"
+             cardTitle="Current Version"
+             i18n-cardTitle
+             aria-label="Current Version"
+             i18n-aria-label
+             id="currentVersion">
+      <div class="d-flex flex-column justify-content-center align-items-center">
+        <h5>{{ version }}</h5>
```

Comment:
Same here: why is the body text so big? There should be some difference between header and body (size, text weight, ...).

![image](https://github.com/ceph/ceph/assets/37327689/5d439962-84b0-41ec-98fa-30db93c36983)

Additionally, can we add some decoration/extra-value to the version? A link to the version release notes? Current URL is:
* `https://docs.ceph.com/en/latest/releases/quincy/#v17-2-6-quincy`

Perhaps we can "discuss" with the docs team to add fixed anchors to those (removing the version code name from the reference, as it makes URL 100% deterministic/future-proof)?